test/onevariable.jl

```diff
@@ -45,6 +45,9 @@ Base.iszero(::SymbNumber) = false
     @test axes(t) == ()
     @test axes([t]) == (Base.OneTo(1),)
 
+    @test 1 > t > t^2
+    @test 1 > -t^2 > -t
```

Comment:
Would this better as `0 >=? -t^2 > -t > 0`?

Reply:
I've reverted the ordering in one of those tests (use `<` or `<=` instead of `>` or `>=`). this is the suggestion, right?